### PR TITLE
Refactor dog model

### PR DESCRIPTION
**Pull request by imfeldd**
Refactor the Dog type and DogCard to use the module structure

### CODE REVIEW

**Comment by imfeldd** [comment]
@SuperQuentin @AlexandrePhilibert @AndiSO-1

**Review by Vidivanze — APPROVED** [review]

File: src/pages/dogs/index.tsx [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+  { Id: "4", Name: "Peppermint", Gender: Gender.Female, Birthdate: new Date("2021-01-01"), BreedName: "Berger allemand", Master: paul, Sterilization: Sterilization.NotSterilized, Color: "", IsDead: false },
 ];
 
 const DogsIndex = () => {
   return <Container maxWidth="lg">
-    { Dogs.map(dog => <DogCard dog={dog}></DogCard>) }
-
-    <Fab aria-label="add" style={{position:"absolute", right: 50, bottom: 50}}>
+    { Dogs.map(dog => <DogCard key={dog.Id} dog={dog}></DogCard>) }
+    <Fab aria-label="add" style={{position:"fixed", right: 50, bottom: 50}}>
```
Comment (AlexandrePhilibertCPNV) [inline src/pages/dogs/index.tsx:21]: use tailwind classes

className="fixed right-12 bottom-12" maybe ?

File: src/pages/services/create.tsx [hunk 2]
```diff
@@ -22,8 +22,8 @@ const Service: NextLayoutPage = () => {
 
   return (
     <Form className="bg-cyan-700 text-white px-5 py-8 h-full" schema={ServiceSchema} onSubmit={handleSubmit}>
-      <h1 className='mb-5'>Création d'un service</h1>
-      <div className='grid grid-cols-12 space-y-3'>
+      <h1 className="mb-5">Création d&aposun service</h1>
```
Comment (AlexandrePhilibertCPNV) [inline src/pages/services/create.tsx:25]: I think using template literals is better, or maybe just a js string with double quotes

 <h1 className="mb-5">"Création d'aposun service"</h1>

File: src/pages/dogs/index.tsx [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
-const paul = new Client("Paul", "Dupont", Gender.Male, "pdupont@cpnv.ch", "+4177123000");
+const paul = { firstname: "Paul", lastname: "Dupont", female: false, email: "pdupont@cpnv.ch", phone: "+4177123000" };
 
-const Dogs = [
-  new Dog("1", "Rufus", Gender.Male, new Date("2020-01-01"), "Berger allemand", paul),
-  new Dog("2", "Duke", Gender.Male, new Date("2018-01-01"), "Berger allemand", paul),
-  new Dog("3", "Boss", Gender.Male, new Date("2021-12-01"), "Berger allemand", paul),
-  new Dog("4", "Peppermint", Gender.Female, new Date("2021-12-01"), "Berger allemand", paul),
+const Dogs: Dog[] = [
+  { Id: "1", Name: "Rufus", Gender: Gender.Male, Birthdate: new Date("2020-01-01"), BreedName: "Berger allemand", Master: paul, Sterilization: Sterilization.NotSterilized, Color: "", IsDead: false },
```
Comment (AlexandrePhilibertCPNV) [inline src/pages/dogs/index.tsx:12]: Attributes in js objects should have a lowercase first letter

**Pull request merged**